File: buggy_script.py
import sys
import fire

def add_numbers(a, b):
    return a + b

def multiply_numbers(a, b):
    return a * b

def divide_numbers(a, b):
    return a / b


def calculate(operation, num1, num2):
    if operation == "add":
        result = add_numbers(num1, num2)
    elif operation == "subtract":
        result = subtract_numbers(num1, num2)
    elif operation == "multiply":
        result = multiply_numbers(num1, num2)
    elif operation == "divide":
        result = divide_numbers(num1, num2)
    else:
        print("Invalid operation")

    return res


if __name__ == "__main__":
    fire.Fire(calculate)

File: buggy_script.js
const subtractNumbers = (a, b) => {
  return a - b;
};

const multiplyNumbers = (a, b) => {
  return a * b;
};

const divideNumbers = (a, b) => {
  return a / b;
};

function calculate(operation, num1, num2) {
  let result = '';
  if (operation == 'add') {
    result = addNumbers(num1, num2);
  } else if (operation == 'subtract') {
    result = subtractNumbers(num1, num2);
  } else if (operation == 'multiply') {
    result = multiplyNumbers(num1, num2);
  } else if (operation == 'divide') {
    result = divideNumbers(num1, num2);
  } else {
    console.log('Invalid operation');
  }

  return res;
}

const [, , operation, num1, num2] = process.argv;
calculate(operation, num1, num2);
